tests/api/v1/generator/base_case.py

```diff
@@ -91,6 +91,7 @@ def _template_data(self):
         return {
             'user': os.environ.get('USER'),
             'workingDirectory': os.environ['PTERO_WORKFLOW_TEST_TEMP'],
+            'environment': json.dumps(dict(os.environ)),
```

Comment:
I didn't think about this before, but should we have an `env` file in each test directory to isolate the environment for the tests?